src/util/well_known_sids.c

```diff
@@ -221,11 +268,96 @@ static errno_t handle_name_to_sid_map(const char *name,
     return EINVAL;
 }
 
+/* This function treats a particular case that does not fit in the normal table */
+static errno_t handle_nt_sids_particular_cases(const char *sid,
+                                               const char **dom,
+                                               const char **name)
+{
+    uint32_t rid;
+    char *endptr;
+    char *startptr;
+    int i;
+
+    rid = (uint32_t) strtouint32(sid + NT_SID_PREFIX_LEN, &endptr, 10);
+    if (errno != 0) {
+        return EINVAL;
+    }
+
+    if (rid == 5) {
+        /* S-1-5-5-x-y, we can ignore x and y, but they must be present */
```

Comment:
I would recommend to add some explanation why the last two components can be ignored, like e.g. the sentence form the MSFT page "The X and Y values for these SIDs are different for each logon session and are recycled when the operating system is restarted."

Reply:
Sure.